signer/src/transaction_coordinator.rs

```diff
@@ -230,24 +229,33 @@ where
         let is_epoch3 = pox_info.current_burnchain_block_height > nakamoto_start_height;
         if is_epoch3 {
             self.is_epoch3 = is_epoch3;
-            tracing::debug!("We are in Epoch 3 or later; time to do work");
+            tracing::debug!("we are in Epoch 3 or later; time to do work");
         }
         Ok(is_epoch3)
     }
 
-    #[tracing::instrument(skip(self))]
+    #[tracing::instrument(skip_all, fields(public_key = %self.signer_public_key(), chain_tip = tracing::field::Empty))]
```

Comment:
suggestion: i'd newline these just for readability

Reply:
https://github.com/stacks-network/sbtc/pull/874/commits/bb02a92a63ab68af348161c9cc504c7788e0654e